### PR TITLE
Added shader support for Vector<Float> and Vector<Int>.

### PR DESCRIPTION
**Pull request by emilwestergren**
Arrays can now be uploaded to shaders by using `Vector`.

### CODE REVIEW

**Comment by emilwestergren** [comment]
@davidpiuva  peer review

File: source/draw/gpu/opengl/OpenGLMap.ooc [hunk 1]
```diff
@@ -95,6 +96,12 @@ OpenGLMap: class extends GpuMap {
 						image _backend bind(textureCount)
 						program setUniform(key, textureCount)
 						textureCount += 1
+					case vector: Vector =>
+						match (vector T) {
+							case Int => program setUniform(key, vector _backend as Int*, vector capacity)
+							case Float => program setUniform(key, vector _backend as Float*, vector capacity)
```
Comment (davidpiuva) [inline source/draw/gpu/opengl/OpenGLMap.ooc]: How do I access this in a shader?

Comment (emilwestergren) [inline source/draw/gpu/opengl/OpenGLMap.ooc]: `uniform int array[20];`

**Comment by davidpiuva** [comment]
Looks good.

**Pull request merged**